src/goto-cc/ms_link_mode.cpp

```diff
@@ -30,7 +30,7 @@ int ms_link_modet::doit()
     return 0;
   }
 
-  eval_verbosity(
+  messaget::eval_verbosity(
     cmdline.get_value("verbosity"), messaget::M_ERROR, message_handler);
 
```

Comment:
Same here

Reply:
I had wondered as well, but yes, it does: the `message_handler` argument will have its verbosity set. We might want to change this interface though as that makes two people who were surprised by this code...